library/std/src/lazy.rs

```diff
@@ -734,7 +717,6 @@ mod tests {
     }
 
     #[test]
-    #[cfg_attr(miri, ignore)] // leaks memory
```

Comment:
Did you verify that Miri does not report leaks any more here?

Reply:
Ah since we are not running these in Miri I guess it doesn't matter... and if we ever do we should re-evaluate this anyway.

Reply:
You did it https://github.com/matklad/once_cell/pull/99 :heart: ^^

Reply:
Ah lol.^^